lib/pantry/helpers/handler_helper.rb

```diff
@@ -0,0 +1,20 @@
+module Pantry
+
+  module Helpers
+
+    class HandlerHelper
+
+      def self.create_response(session_attributes: {}, message: , end_session: true)
+        response = AlexaRubykit::Response.new
+        response.add_speech message.to_s
+        session_attributes.each do |k, v|
```

Comment:
I see you using this `session_attributes` key in the tests for this class, but nowhere else. Throwing it in thinking that it'll be useful later?